scripts/import_standard_ebooks.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+import json
+import requests
+import sys
+import time
+
+import os.path as path
+
+import feedparser
+
+from openlibrary.core.imports import Batch
+
+sys.path.insert(0, path.abspath(path.join(path.sep, 'openlibrary')))
+from openlibrary.config import load_config  # noqa: E402
+load_config(
+    path.abspath(path.join(
+        path.sep, 'olsystem', 'etc', 'openlibrary.yml')))
+from infogami import config  # noqa: E402,F401
```

Comment:
@mekarpeles I don't remember if I was able to test this properly on my local instance.  Maybe we can use `scripts/_init_path.py` if this isn't working properly.